src/screens/MessageScreen.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Image, FastImageSource } from 'components/Image';
+import { ButtonProps } from 'react-native-elements';
+import { Text, View, StyleSheet } from 'react-native';
+import { useNavigationParam } from 'react-navigation-hooks';
+import { Button } from 'components/Button';
+
+import { typography, palette } from 'styles';
+
+export const MessageScreen = () => {
+  const title: string = useNavigationParam('title');
+  const source: FastImageSource = useNavigationParam('source');
+  const description: string = useNavigationParam('description');
+  const button: ButtonProps | undefined = useNavigationParam('button');
```

Comment:
Maybe you know if there's a possibility to make these params required?

Example of usage:
```
  NavigationService.navigate('Message', {
            title: 'Success',
            description: 'Your contatct has been successfully deleted. You can now return to Address book.',
            source: images.success,
            button: { title: 'Return to Address book' }
          })
```
In that case we can omit the param and we don't have an error :thinking: